src/main/java/com/laytonsmith/abstraction/enums/MCChatColor.java

```diff
@@ -124,6 +124,17 @@ public static MCChatColor getByChar(char code) {
 		return CHAR_COLORS.get(code);
 	}
 
+	public static String fromRGBValue(String hexString) {
+		if(hexString.length() > 7) {
+			hexString = hexString.substring(0, 7);
```

Comment:
In this case, `null` should be returned as the string isn't a valid RGB/hex color identifier. Otherwise, `color('#FFFFFFblabla')` would be accepted.

Reply:
I did it this way because `color()` does it for everything else, where it ignores extra stuff after it recognises a colour. ie `color('abcde')` will still become the 'a' code.